app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/fragment/UserBookFragment.java

```diff
@@ -287,4 +296,20 @@ private void onClickButton() {
     private void onClickTempButton() {
 
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+
+        // Check which request we're responding to
+        if (requestCode == RESULT_ISBN) {
+            // Make sure the request was successful
+            if (resultCode == RESULT_OK) {
+                Bundle extras = data.getExtras();
+                String isbn = extras.getString("isbn");
+                Book book = ISBNLookup.searchISBN(isbn);
```

Comment:
This is UI thread. This may cause the program to freeze

Reply:
AsyncTask doesnt run on UI thread